easybuild/easyconfigs/x/XZ/XZ-5.2.3-CrayGNU-2016.11.eb

```diff
@@ -0,0 +1,31 @@
+# contributed by Luca Marsella (CSCS)
+easyblock = 'ConfigureMake'
+
+name = 'XZ'
+version = "5.2.3"
+
+homepage = 'http://tukaani.org/xz/'
+description = "xz: XZ utilities"
+
+toolchain = {'name': 'CrayGNU', 'version': '2016.11'}
```

Comment:
2016.11 still needed  ?

Reply:
Not really, it was an intermediate file before creating the final recipe.